src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs

```diff
@@ -181,6 +181,16 @@ public static NativeInterop.AuthParameters GetCommonAuthParameters(
             {
                 authParams.Properties["instance_aware"] = "true";
             }
+            
+            //SSH Cert
+            if(authenticationRequestParameters.AuthenticationScheme.AccessTokenType == "ssh-cert")
```

Comment:
>  if(authenticationRequestParameters.AuthenticationScheme.AccessTokenType == "ssh-cert") [](http://example.com/codeflow?start=10&length=88)

Should this be a case-insensitive check ?

Reply:
From what I've seen, the AccessTokenType is always lowercase